README.md

```diff
@@ -71,6 +71,10 @@ This should be updated fairly regularly. As usual, **pull requests are encourage
 * [FastAPI](https://fastapi.tiangolo.com) - Python framework for building APIs with beautiful and complete documentation. (contributed by [@jrwaine](https://github.com/jrwaine))
 * [Circles, Sines and Signals](https://jackschaedler.github.io/circles-sines-signals/) - A Compact Primer on Digital Signal Processing. (contributed by [@PharkMillups](https://github.com/PharkMillups))
 * [PostgreSQL](https://www.postgresql.org/docs/current/index.html) - One of the most reliable sets of documentation available. There is rarely a need to go outside the documentation. (contributed by [@dalryan](https://github.com/dalryan))
+* [Svelte](https://svelte.dev/docs/introduction) - Svelte shifts as much work as possible out of the browser and into your build step. No more manual optimisations — just faster, more efficient apps. (contributed by [@0Ared](https://github.com/0Ared/))
+* [Svelte Kit](https://kit.svelte.dev/docs/introduction) - SvelteKit gives you the tools to succeed whatever it is you're building. And it runs wherever JavaScript does.  (contributed by [@0Ared](https://github.com/0Ared/))
+
+
```

Comment:
I don't think it's necessary to add these two sources. They have the same structure/design and you can reach kit.svelte.dev through the navigation menu of svelte.dev. Could you remove them, please?

```suggestion
```